src/utils/fetch.js

```diff
@@ -61,7 +61,12 @@ export function convertFetchResponse(url, body, result, isRejected) {
             result,
         });
 
-        throw result;
+        const networkError = {
+            message: result.message,

```

Comment:
how do you know there is a result.message? on every browser? I suggest result || result.message